### PR TITLE
fix travis

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by fpietka** [comment]
@keleshev Seems safe to be merged, as travis gives an error on latest builds:
`The command "pip install tox --use-mirrors" failed and exited with 2 during .`

**Comment by graingert** [comment]
@mboersma any chance of pulling this in?

**Comment by keleshev** [comment]
I don't understand what was the reason it was failing and how this fixed it—could you explain?

Note, I dislike the fact that the list of python versions is now duplicated across two files.

**Comment by graingert** [comment]
> Note, I dislike the fact that the list of python versions is now duplicated across two files.

One is a list for tox (for local users) the other is to make them parallel in travis.

**Comment by graingert** [comment]
@keleshev `--use-mirrors` just breaks things.

**Comment by fpietka** [comment]
I might be wrong, but it may be related to travis having changed (upgraded) their build images.

**Comment by graingert** [comment]
@fpietka yes, the flag was removed from pip because it was pointless.

**Comment by benjaoming** [comment]
@keleshev @mboersma should probably merge this important sustainability contribution to the repo :)

**Comment by mboersma** [comment]
I apologize for letting this dangle so long.

It looks like `TOX_ENV=py26` is [no longer supported](https://travis-ci.org/docopt/docopt/jobs/244990261) on Travis CI. I'll merge this and then remove Python 2.6 from the CI matrix so we can get green again. Thanks @graingert!

**Comment by mboersma** [comment]
Ok, now I'm confused, since post-merge the [master build](https://travis-ci.org/docopt/docopt/builds/326080359) actually failed to find py26, py33, and pypy.

**Comment by graingert** [comment]
py33 is dead also

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,16 @@
-# Travis (http://travis-ci.org/) is a continuous integration
-# service for open source projects. This file configures
-# Travis to install and run "tox" test runner, which is
-# configured in tox.ini file.
-
+sudo: false
 language: python
-install: pip install tox --use-mirrors
-script: tox
+python: 3.5
```
Comment (benjaoming) [inline .travis.yml]: I would advice to specify all Python versions used here, and then use tox-travis to resolve the active Python version. Example: https://github.com/django-wiki/django-wiki/blob/master/tox.ini

**Comment by hugovk** [comment]
I think it's safe enough to drop support for Python 2.6 and 3.3, they're EOL and no longer receive security updates, and are little used.

image

Here's the pip installs for docopt from PyPI for the last month (via `pypinfo --percent --pip --markdown docopt pyversion`), showing low fractions for 2.6 and 3.3:

| python_version | percent | download_count |
| -------------- | ------: | -------------: |
| 2.7            |   77.5% |        961,861 |
| 3.6            |    9.7% |        120,364 |
| 3.5            |    7.0% |         87,098 |
| 3.4            |    4.3% |         53,284 |
| 2.6            |    0.6% |          7,966 |
| 3.3            |    0.5% |          5,592 |
| 3.7            |    0.4% |          4,736 |
| 3.2            |    0.1% |            956 |
| None           |    0.0% |              1 |

**Pull request merged**